pkg/solana/chain.go

```diff
@@ -221,6 +229,61 @@ func newChain(id string, cfg *config.TOMLConfig, ks loop.Keystore, lggr logger.L
 		lggr:        logger.Named(lggr, "Chain"),
 		clientCache: map[string]*verifiedCachedClient{},
 	}
+
+	if cfg.MultiNodeEnabled() {
+		chainFamily := "solana"
+
+		mnCfg := cfg.MultiNodeConfig()
+
+		var nodes []mn.Node[mn.StringID, *client.Client]
+
+		for i, nodeInfo := range cfg.ListNodes() {
+			// create client and check
+			rpcClient, err := client.NewClient(nodeInfo.URL.String(), cfg, DefaultRequestTimeout, logger.Named(lggr, "Client."+*nodeInfo.Name))
+			if err != nil {
+				lggr.Warnw("failed to create client", "name", *nodeInfo.Name, "solana-url", nodeInfo.URL.String(), "err", err.Error())
+				continue
```

Comment:
Nit: returning an error instead of just logging might be a good idea. CL Node is not functional if we fail to create all clients, so it should not start